mirage/config.ts

```diff
@@ -316,6 +316,12 @@ export default function(this: Server) {
     });
     this.post('/search/collections/', searchCollections);
 
+    osfResource(this, 'output', { path: '/outputs' , only: ['create', 'show', 'update']});
```

Comment:
When we close the "Add output/resource" modal, are we going to delete the non-finalized `output`?

Reply:
Yeah, I'll change this to `except: ['index']`.